content/en/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -570,7 +570,7 @@ kube-apiserver [flags]
       <td colspan="2">--etcd-servers-overrides stringSlice</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Per-resource etcd servers overrides, comma separated. The individual override format: group/resource#servers, where servers are URLs, semicolon separated.</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">Per-resource etcd servers overrides, comma separated. The individual override format: group/resource#servers, where servers are URLs, semicolon separated. Note that etcd servers overrides currently do not support CRDs.</td>
```

Comment:
Hello @droccom ,
This file is generated from the upstream kube-apiserver source code.
To make this change permanent, you should create this change in kubernetes/kubernetes.

Reply:
Thanks, will do.

Reply:
👋 @droccom,
If you are interested, you can add this change now (reopen the pull request).
Otherwise, the next time this page is generated your upstream changes will be incorporated into the page.